components/wpt-results.html

```diff
@@ -248,64 +218,70 @@
       }
 
       _queryChanged () {
-        this._refreshDisplayedSpecDirs()
+        this._refreshDisplayedNodes()
       }
 
-      _refreshDisplayedSpecDirs () {
+      _refreshDisplayedNodes () {
         /* Recomputes the list of displayed spec directories and their child test results. */
```

Comment:
Done. It can be either - directories and paths are displayed intermixed with each other now. Rephrased to `Recomputes the list of displayed directories and test files.`.